src/storage/src/controller.rs

```diff
@@ -7,6 +7,8 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+#![allow(missing_docs)]
```

Comment:
naughty :sweat_smile:

Reply:
It was there before, just in the other module!